mcs/class/monodoc/Monodoc/providers/EcmaDoc.cs

```diff
@@ -578,16 +602,19 @@ internal static string MakeOperatorSignature (XElement member, out string member
 
 		static XElement ExtractClassSummary (XDocument typeDoc)
 		{
-			string name = typeDoc.Root.Attribute("Name").Value;
-			string fullName = typeDoc.Root.Attribute("FullName").Value;
-			string assemblyName = typeDoc.Root.Element("AssemblyInfo") != null ? typeDoc.Root.Element("AssemblyInfo").Element("AssemblyName").Value : string.Empty;
-			var docs = typeDoc.Root.Element("Docs");
-			var summary = docs.Element("summary") ?? new XElement("summary");
-			var remarks = docs.Element("remarks") ?? new XElement("remarks");
+			string name = typeDoc.Root.Attribute ("Name").Value;
```

Comment:
The problem with whitespace changes is that they pollute history. You can't "just" do `git blame` to see when the line was added, because now the line was changed for purely whitespace reasons.

Thus, instead of:

    git blame file.cs

it's:

    git blame file.cs
    # get commit of line change; see it's just a whitespace change
    git blame COMMIT^ file.cs
    # repeat

and if `file` was ever MOVED, then `git blame COMMIT^ file.cs` fails because the path is wrong, and this is MADNESS.

Whitespace changes are evil if you ever need to dig through code history to figure out *why* a change was done.

> Is it verboten to do whitespace changes ever?

Let's go with "yes". You can do whitespace changes *in addition to other changes which impact the same line*, e.g. if a variable name changes and thus the line would change *anyway*, then additional whitespace changes to that line would be fine. (Though because of the complications in viewing history, renaming a variable "just because" is also something to be discouraged.)